sidebars.js

```diff
@@ -493,6 +493,7 @@ module.exports = {
         "references/configuration",
         "references/ui-configuration",
         "references/server-options",
+        "references/docker-env-vars",
```

Comment:
I'll make a case for moving this link up one line so that it appears immediately following the Web UI reference. @flossypurse?

Reply:
ya, put this below the configuration

And we should have some explanation about the relationship between the 2 explicitly - and any other context setting information we can scrounge up.